### PR TITLE
Fix #1351: add maintenance mode

### LINKED ISSUE #1351: Maintenance mode
Install & configure https://github.com/lexik/LexikMaintenanceBundle

### PR DESCRIPTION
**Pull request by nicosomb**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | yes
| Translation   | no
| Fixed tickets | #1351 
| License       | MIT

~~We need to wait for this PR before merging https://github.com/lexik/LexikMaintenanceBundle/pull/52~~

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -64,7 +64,8 @@
         "guzzlehttp/guzzle": "^5.2.0",
         "doctrine/doctrine-migrations-bundle": "^1.0",
         "paragonie/random_compat": "~1.0",
-        "craue/config-bundle": "~1.4"
+        "craue/config-bundle": "~1.4",
+        "lexik/maintenance-bundle": "dev-master"
```
Comment (delyriand) [inline composer.json]: think to use a tag instead of dev-master

Comment (j0k3r) [inline composer.json]: +1

Comment (nicosomb) [inline composer.json]: +1 ... But can't do that. The bundle doesn't respect semver ...

Comment (nicosomb) [inline composer.json]: we need to wait for a new release

Comment (j0k3r) [inline composer.json]: Fixed https://github.com/lexik/LexikMaintenanceBundle/releases/tag/v2.1.0

**Pull request merged**